templates/src/util/gsap-animate.js

```diff
@@ -5,8 +5,7 @@
  * import animate from '../../util/gsap-animate';
  * animate.to(this.container, 0.3, { autoAlpha: 1, delay: 0.2 }).then(()=>{});
  */
-import { TweenLite, CSSPlugin } from 'gsap';
-import { ScrollToPlugin } from 'gsap/src/uncompressed/plugins/ScrollToPlugin';
+import { TweenLite, CSSPlugin, ScrollToPlugin } from 'gsap/all';
```

Comment:
as per this thread, `gsap/all` is a preferred way to import https://greensock.com/forums/topic/18437-import-from-gsap-vs-from-gsapall/

Reply:
@iranreyes oh `react-scripts test` breaks on import from `gsap/all`